Libraries/Lists/VirtualizedSectionList.js

```diff
@@ -202,6 +168,30 @@ class VirtualizedSectionList<SectionT: SectionBase> extends React.PureComponent<
     );
   }
 
+  scrollToLocation(params: {
```

Comment:
Please avoid unnecessary changes - makes it harder to review. If you think this should be moved here, please submit a separate PR.